homeassistant/components/light/abode.py

```diff
@@ -68,7 +73,8 @@ def is_on(self):
     def brightness(self):
         """Return the brightness of the light."""
         if self._device.is_dimmable and self._device.has_brightness:
-            return self._device.brightness
+            # Convert Abode brightness (0-100) to HASS brightness (0-255)
+            return int(self._device.brightness) * 255 / 100
```

Comment:
Does the device return 100% ever here? I assume the upper limit should be 99 again.

Reply:
The max value I see returned is 99 from Abode. Similar to my comment above, would something like this be better suited?

````
brightness = int(max(min(self._device.brightness, 99), 0) * 255 / 100)
return brightness
````